src/styles/index.scss

```diff
@@ -1,3 +1,73 @@
+$cell-width: 100px;
+$cell-height: 100px;
+$days: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  flex-wrap: wrap;
+  gap: 1px;
+  width: calc(7 * $cell-width + 6 * 1px);
+  padding: 0 10px;
+  display: flex;
+
+  &__day {
+    background-color: #eee;
+    width: $cell-width;
+    height: $cell-height;
+    border: 1px solid black;
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: transform 500ms;
+    cursor: pointer;
+  }
+  &__day:hover {
+    background-color: #ffbfcb;
+    transform: translateY(-20px);
```

Comment:
For consistency and maintainability, consider using a variable for the 'transform' value that is likely to be reused, similar to how you've used variables for dimensions.